prelude.dx

```diff
@@ -174,15 +174,9 @@ data Bool =
   False
   True
 
-def BToW8 (x : Bool) : Word8 =
-  case x of
-    False -> (IToW8 0)
-    True  -> (IToW8 1)
+def BToW8 (x : Bool) : Word8 = %dataConTag x
```

Comment:
Do you do that to make the size of the code we emit smaller?

Reply:
Honestly this should just be an optimization we apply to Core/Imp. Not a new prim op.

Reply:
Yes, it makes a meaningful difference to code size because otherwise we generate branches every time we write `&&` or `||`. We can remove it when our core/imp optimizations can do it.